python_anvil/api_resources/payload.py

```diff
@@ -31,9 +31,10 @@ class FillPDFPayload(DataClassJsonMixin):
 @dataclass_json(letter_case=LetterCase.CAMEL)
 @dataclass
 class GeneratePDFPayload(DataClassJsonMixin):
-    data: List[Dict[str, Any]]
+    data: Union[List[Dict[str, Any]], Dict[str, Any]]
```

Comment:
I know nothing of Python typing, but is `Union` the right wrapper here? It accomplishes an `OR`, essentially?

Reply:
Yeah, basically [(docs)](https://docs.python.org/3/library/typing.html#typing.Union). In other languages it would be like `TypeA | TypeB`.